src/bootstrap/src/core/build_steps/compile.rs

```diff
@@ -1541,7 +1541,8 @@ pub fn compiler_file(
         return PathBuf::new();
     }
     let mut cmd = command(compiler);
-    cmd.args(builder.cflags(target, GitRepo::Rustc, c));
+    cmd.args(builder.default_cflags(target, c));
+    cmd.args(builder.extra_cflags(target, GitRepo::Rustc, c));
```

Comment:
Discussion: based on the method comments, should we just be explicit and call them `cc_handled_clags` vs `cc_unhandled_cflags`?

Reply:
That's fine by me!